pandas/core/nanops.py

```diff
@@ -1288,7 +1289,11 @@ def _ensure_numeric(x):
             try:
                 x = x.astype(np.complex128)
             except (TypeError, ValueError):
-                x = x.astype(np.float64)
+                try:
```

Comment:
can you document this at some point. also likely this should be merged put in pandas.core.dtypes.common (e.g. the ensure_* functions)

Reply:
yah, i keep meaning to do a few passes to clean up nanops, maybe after the current indexing kick